src/uwloc/data/main.py

```diff
@@ -8,8 +9,12 @@
 from .tile import MAX_HOURS, MAX_UNITS, create, init_tiledb, tidy, write_row
 from .wav import read_wav, read_wav_metadata
 
-app = typer.Typer()
 WAV_PATTERN = "**/*.[wW][aA][vV]"
+app = typer.Typer()
+logger = logging.getLogger(__name__)
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s %(levelname)s %(module)s %(funcName)s %(message)s"
+)
```

Comment:
NIT: For readability, can you put logging stuff before app and put a space between them? That way it's easier to find the app object for typer, plus logging is using Python builtins and typer is 3rd party, make it similar rules as importing 😉